### PR TITLE
Integration Flight Test 12

### PR DESCRIPTION
**Pull request by mbenson1**

### CODE REVIEW

File: config/bebop2/sitl/host/inc/cfe_platform_cfg.h [hunk 1]
```diff
@@ -1354,7 +1354,7 @@
 **  \par Limits
 **       There is an upper limit of 32767 on this configuration paramater.
 */
-#define CFE_EVS_MAX_EVENT_FILTERS     255
+#define CFE_EVS_MAX_EVENT_FILTERS     256
```
Comment (ldoman) [inline config/bebop2/sitl/host/inc/cfe_platform_cfg.h]: Modifications to the cfe platform config have no impact unless made in the core-builder repo. Is this just syncing with what the core-builder has?

File: config/ocpoc/default/target/cfe_es_startup.scr [hunk 2]
```diff
@@ -28,11 +30,13 @@ CFE_APP, /cf/apps/VM.so,        VM_AppMain,      VM,       70, 327680, 0x0, 0, 4
 CFE_APP, /cf/apps/CS.so,        CS_AppMain,      CS,        160, 327680, 0x0, 0, 2;
 CFE_APP, /cf/apps/RCIN.so,      RCIN_AppMain,    RCIN,      88, 327680, 0x0, 0, 2;
 CFE_APP, /cf/apps/LD.so,        LD_AppMain,      LD,       73, 327680, 0x0, 0, 4;
+CFE_APP, /cf/apps/VC.so,        VC_AppMain,      VC,       181, 327680, 0x0, 0, 2;
+CFE_APP, /cf/apps/EA.so,        EA_AppMain,      EA,        184, 327680, 0x0, 0, 2;
```
Comment (ldoman) [inline config/ocpoc/default/target/cfe_es_startup.scr]: It looks like EA was added here twice

File: config/ocpoc/quadx/target/commander/commander_workspace/etc/udpstdprovider-config.json [hunk 3]
```diff
@@ -35,7 +35,7 @@
 {
     inPort:  5011,
     outPort: 5010,
-    outAddress: '192.168.1.150',
+    outAddress: '192.168.1.66',
```
Comment (ldoman) [inline config/ocpoc/quadx/target/commander/commander_workspace/etc/udpstdprovider-config.json]: Is outAddress where CDR is sending udp commands to? This used to point to 150 which is the dev ocpoc, and its been changed to 66 which is my laptop's ip.

File: config/bebop2/sitl/host/inc/cfe_platform_cfg.h (hunk 1 above)
Comment (mbenson1) [reply config/bebop2/sitl/host/inc/cfe_platform_cfg.h]: Fixing core-builder.

File: config/ocpoc/default/target/cfe_es_startup.scr (hunk 2 above)
Comment (mbenson1) [reply config/ocpoc/default/target/cfe_es_startup.scr]: Fixed

File: config/ocpoc/quadx/target/commander/commander_workspace/etc/udpstdprovider-config.json (hunk 3 above)
Comment (mbenson1) [reply config/ocpoc/quadx/target/commander/commander_workspace/etc/udpstdprovider-config.json]: Fixed

**Pull request merged**